Dockerfile

```diff
@@ -1,3 +1,19 @@
+# Copyright 2025 nvidia-k8s-ipam authors
```

Comment:
Let's keep `NVIDIA CORPORATION & AFFILIATES` for this project.
No external contribution until now